<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(216, 216, 216);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>18</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 86, 56);</string>
    </property>
    <property name="text">
     <string>CONFIGURE OSCILLOSCOPE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>134</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Instrument</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>204</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel In</string>
    </property>
   </widget>
   <widget class="QComboBox" name="channelInBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>CHANNEL1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CHANNEL2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CHANNEL3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CHANNEL4</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="continueButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CONTINUE</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="probeInCheck">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>x10 Probe</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="probeOutCheck">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>x10 Probe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>276</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel Out</string>
    </property>
   </widget>
   <widget class="QComboBox" name="channelOutBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentText">
     <string>CHANNEL2</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>CHANNEL1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CHANNEL2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CHANNEL3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CHANNEL4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QComboBox" name="visaStringOsc">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
